### PR TITLE
fix(Core): Crashfix.

### LINKED ISSUE #11296: [server] wordserver crash
### Current Behaviour

crash log
https://gist.github.com/mpfans/0ef2ba09e560a29d73f2555d0ccfc27a

### Expected Blizzlike Behaviour

none

### Source

_No response_

### Steps to reproduce the problem

none

### Extra Notes

_No response_

### AC rev. hash/commit

285504e79d25c84b88413fdfad17c2b83fefda86

### Operating system

ubuntu 20.04

### Custom changes or Modules

_No response_

### PR DESCRIPTION
**Pull request by UltraNix**
## Issues Addressed:

- Closes #11296

## Tests Performed:

- Not tested.

## How to Test the Changes:

Nothing to be tested.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by UltraNix** [comment]
@mpfans Please, test it.

File: src/server/game/AuctionHouse/AuctionHouseMgr.cpp [hunk 1]
```diff
@@ -731,6 +731,9 @@ bool AuctionHouseObject::BuildListAuctionItems(WorldPacket& data, Player* player
 {
     uint32 itrcounter = 0;
 
+    // Ensures that listfrom is not greater that auctions count
+    listfrom = std::min(listfrom, static_cast<uint32>(GetAuctions().size()));
```
Comment (Kitzunu) [inline src/server/game/AuctionHouse/AuctionHouseMgr.cpp:735]: where is this used?

Comment (UltraNix) [reply src/server/game/AuctionHouse/AuctionHouseMgr.cpp:735]: Just scroll the code down.

Comment (Kitzunu) [reply src/server/game/AuctionHouse/AuctionHouseMgr.cpp:735]: my brain being slow. sorry

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**